### PR TITLE
Changed Build Version to 1.8.

### PR DESCRIPTION
**Pull request by Rieprecht**
Since we uses Lambda Expressions in SubmitController we need Java 1.8.
Furthermore test-scope of junit in POM removed.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -170,7 +170,6 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.9</version>
                 <configuration>
-                    <skip>${maven.skip-frontend}</skip>
```
Comment (Fabsi110) [inline pom.xml]: Is this correct?

**Comment by Fabsi110** [comment]
I don't see an update to 1.8 in the changes.

**Comment by Rieprecht** [comment]
You are right, no change to 1.8... curious. However it's now 1.8 and that's fine.

**Comment by MonsJovis** [comment]
Was is nun damit?

**Comment by Rieprecht** [comment]
Since the version was changed in a prior commit and the junit Problem (test-scope) was solved in an other manner this PR can be closed.

**Pull request closed without merging**